Source/WPF/MyMoney/Database/CsvStore.cs

```diff
@@ -151,77 +148,64 @@ public virtual DataSet QueryDataSet(string cmd)
             return new DataSet();
         }
 
-        private static string[] ParseCSVLine(string csvLine)
-        {
-            using (TextFieldParser TFP = new TextFieldParser(new MemoryStream(Encoding.UTF8.GetBytes(csvLine))))
-            {
-                TFP.HasFieldsEnclosedInQuotes = true;
-                TFP.SetDelimiters(",");
-                return TFP.ReadFields();
-            }
-        }
+
+        /// <summary>
+        /// Import the Transactions & accounts in the given file and return the first account.
+        /// </summary>
+        /// <param name="file"></param>
+        /// <param name="count"></param>
+        /// <returns></returns>
 
         public static int ImportCsv(MyMoney myMoney, Account acct, string file)
         {
+            var uri = new Uri(file);
+            var csvReader = new CsvReader(uri, Encoding.Default, null, 4096);

```

Comment:
What is Encoding.Default on your machine  - is it UTF-8  and is that what your bank says they export?  You could test if you have any special European chars in the names of things then look at the file and make sure it is utf-8 encoding those chars...

Reply:
For now yes, its all basic ascii char and the data files that I passing before are still passing with your csvReader

Reply:
For example, suppose you have a Payee named "compañero" then the CSV file should contain:
```
63 6f 6d 70 61 c3 b1 65 72 6f                     compaA.ero
```

Reply:
Agree, I just have not hit that use case. We can add a different Encoding type. But I need to test with a real world data. I do not have this at the moment

Reply:
So you don't have anything from your bank containing special non-ascii (0-127) chars?  If so, then we'll have to debug this later if you run into any issues, which is fine.  I'm happy with UTF-8 as the default although I'd be inclined to hard code it to `Encoding.UTF8` in case there's some way windows can mess with the Encoding.Default.

Reply:
I made the change Encoding.UTF8, now how to I update this Pull Request?  Do I create a new Pull Request for that 1 line change?

Reply:
Looks good.

Reply:
Ship it!